frontend/src/pages/devices/List.tsx

```diff
@@ -85,9 +80,9 @@ export const List = () => {
                     {entities.map(entity =>
                         <tr key={ entity.id }>
                             <th scope="row">
-                                <Button variant="link" onClick={evt => labelClick(entity.id)}>
-                                    { entity.rawDevice.name }
-                                </Button>
+                                <Link to={"/devices/" + entity.id}>
```

Comment:
Please add a space after the `{` and before the `}` to make it beautiful. You can include that change in a future PR.